tests/test_image.py

```diff
@@ -173,6 +174,16 @@ def test_heic(self):
         self.assertEqual(height, 241)
         self.assertEqual(image.mime_type, "image/heiс")
 
+    def test_avif(self):
+        with open("tests/images/tree-avif.avif", "rb") as f:
```

Comment:
```suggestion
        with open("tests/images/tree.avif", "rb") as f:
```

Reply:
would this work on `WandImage` ?? I think it requires `PIL.Image`?